graphs/frequent_pattern_graph_miner.py

```diff
@@ -0,0 +1,189 @@
+'''
+FP-GraphMiner - A Fast Frequent Pattern Mining Algorithm for Network Graphs
+
+A novel Frequent Pattern Graph Mining algorithm,
+FP-GraphMiner, that compactly represents a set of network graphs as a Frequent Pattern Graph (or FP-Graph).
+This graph can be used to efficiently mine frequent subgraphs including maximal frequent subgraphs and maximum common subgraphs.
+
+URL:https://www.researchgate.net/publication/235255851_FP-GraphMiner_-_A_Fast_Frequent_Pattern_Mining_Algorithm_for_Network_Graphs
+
+'''
+def get_distinct_edge(edge_array):
+    '''
+    Return Distinct edges from edge array of multiple graphs
+    '''
+    distinct_edge=set()
+
+    for i in enumerate(edge_array):    
+        for j in enumerate(i[1]):

```

Comment:
enumerate returns two values: an integer index and a row of elements...
```suggestion
    for i, row in enumerate(edge_array):    
        for j, item in enumerate(???):
```
```